scripts/cmake/vcpkg_build_make.cmake

```
@@ -142,15 +142,21 @@ function(vcpkg_build_make)
             z_vcpkg_extract_cpp_flags_and_set_cflags_and_cxxflags("${cmake_buildtype}")
 
             if(VCPKG_LIBRARY_LINKAGE STREQUAL "static")
-                set(LINKER_FLAGS_${cmake_buildtype} "${VCPKG_DETECTED_STATIC_LINKER_FLAGS_${cmake_buildtype}}")
+                set(LINKER_FLAGS_${cmake_buildtype} "${VCPKG_DETECTED_CMAKE_STATIC_LINKER_FLAGS_${cmake_buildtype}}")
             else() # dynamic
-                set(LINKER_FLAGS_${cmake_buildtype} "${VCPKG_DETECTED_SHARED_LINKER_FLAGS_${cmake_buildtype}}")
+                set(LINKER_FLAGS_${cmake_buildtype} "${VCPKG_DETECTED_CMAKE_SHARED_LINKER_FLAGS_${cmake_buildtype}}")
+            endif()            
+            set(LDFLAGS_${cmake_buildtype} "")
+            if(EXISTS "${Z_VCPKG_INSTALLED}${path_suffix}/lib")
+                string(APPEND LDFLAGS_${cmake_buildtype} " -L${Z_VCPKG_INSTALLED}${path_suffix}/lib")
+            endif()
+            if(EXISTS "${Z_VCPKG_INSTALLED}${path_suffix}/lib/manual-link")
+                string(APPEND LDFLAGS_${cmake_buildtype} " -L${Z_VCPKG_INSTALLED}${path_suffix}/lib/manual-link")
             endif()

```

This is clearly an improvement vs. the status quo so I'm merging it, however, shouldn't this be:

```suggestion
            if(EXISTS "${Z_VCPKG_INSTALLED}${path_suffix}/lib")
                string(APPEND LDFLAGS_${cmake_buildtype} " -L\"${Z_VCPKG_INSTALLED}${path_suffix}/lib\"")
            endif()
            if(EXISTS "${Z_VCPKG_INSTALLED}${path_suffix}/lib/manual-link")
                string(APPEND LDFLAGS_${cmake_buildtype} " -L\"${Z_VCPKG_INSTALLED}${path_suffix}/lib/manual-link"")
            endif()

```

or something like that? (I'm not sure what syntax is is strictly speaking correct here but I suspect we should be defending against spaces)

---

It looks like the diff is confused. All I did was add quotes, changing:

```
"-L${Z_VCPKG_INSTALLED}${path_suffix}/lib/manual-link"
```

to

```
"-L\"${Z_VCPKG_INSTALLED}${path_suffix}/lib/manual-link\""
```